backend/README.md

```diff
@@ -65,25 +65,140 @@ By completing these steps, the backend will be primed with the essential verifie
+
+In this example, a test environment is set up with the anvil instance by invoking the `initialize_test_env` method. This environment is also utilized in other examples such as `submit_solvency` and `verify_inclusion_on_contracts`.
+
+Key points to note:
+
+The instance of `AddressOwnership` is initialized with `signatures.csv`, and is named `address_ownership_client`. This instance has already loaded the signature data.
+
+The `dispatch_proof_of_address_ownership` function sends a transaction to the Summa contract, registering the addresses owned by the CEX on the contract.
+
+After dispatching the transaction via `dispatch_proof_of_address_ownerhip`, the example computes the hashed addresses (address_hashes) to verify they have been correctly registered on the Summa contract.
+
+To excute this example:
+```
+cargo run --example submit_ownership
+```
+
+Upon successful execution, you should see the message:
+```
+Ownership proofs are submitted successfully!
+```
+
+Reminder: This demonstration takes place in a test environment. In real-world production, always ensure that the Summa contract is correctly deployed on the target chain.
+
+### 3. Submit Proof of Solvency
+
+Before generate inclusion proof for every user of the current round, You should submit proof of sovlency to Summa contract. Currently, we made this as mandatory way to commit the root hash of the Merkle Sum Tree.
+
+Without this process, It seems the user may not trust to the inclusion proof for the round. becuase the `mst_root` is not published on contract. More specifically, it means that the `mst_root` is not correctly verified on the Summa contract.
+
+In this example, we'll guide you through the process of submitting a solvency proof using the Round to the Summa contract.
+The Round serves as the core of the backend in Summa, and we have briefly described it in the Components section.
+
+To initialize the Round, several parameters are required, including paths to specific CSV files (`assets.csv` and `entry_16.csv`), as well as a path to the ptau file (`ptau/hermez-raw-11`).
+
+The roles of these files are as follows:
+- `assets.csv`: This file is essential for calculating the total balance of assets for the solvency proof. Currently, only the CEX can generate this asset CSV file in its specific manner.
+
+- `entry_16.csv`: This file is used to build the Merkle sum tree, where each leaf element originates from sixteen entries in the CSV.
+X
+- `ptau/hermez-raw-11`: Contains the Powers of Tau trusted setup parameters, essential for constructing the zk circuits.
+
+By employing the round instance, the solvency proof is dispatched to the Summa contract using the `dispatch_solvency_proof` method.
```

Comment:
"An instance of `Round` dispatches the solvency proof using the `dispatch_solvency_proof` method."

Reply:
Replaced at [eacfe7d](https://github.com/summa-dev/summa-solvency/pull/159/commits/eacfe7df0542f35ab9367cc34ad8a298b0cdda5d)